modules/integration/tests-common/integration-tests-utils/src/main/java/org/wso2/mb/integration/common/utils/ui/pages/main/DLCContentPage.java

```diff
@@ -54,6 +55,36 @@ public String deleteFunction() throws IOException {
     }
 
     /**
+     * Delete all the messages in DeadLetter Channel
+     */
+    public void deleteAllDLCMessages() {
+
+        String deletingMessageID = null;
+
+        try {
+            deletingMessageID = driver.findElement(By.xpath(UIElementMapper.getInstance()
+                                                     .getElement("mb.dlc.first.message.id"))).getText();
+            if(StringUtils.isNotBlank(deletingMessageID)) {
+                log.info("delete all dlc messages");
+                driver.findElement(By.xpath(UIElementMapper.getInstance()
+                                                    .getElement("mb.dlc.browse.table.choose.all.box.xpath")))
+                                                    .click();
+                driver.findElement(By.xpath(UIElementMapper.getInstance()
+                                                    .getElement("mb.dlc.browse.table.delete.button"))).click();
+                driver.findElement(By.xpath(UIElementMapper.getInstance()
+                                                    .getElement("mb.dlc.browse.function.confirm"))).click();
+                driver.findElement(By.xpath(UIElementMapper.getInstance()
+                                                    .getElement("mb.dlc.browse.function.success"))).click();
+            }
+
+        }catch (Exception e) {
+            log.error("unable to find any valid messages in dead letter channel.");
```

Comment:
I suppose this UI part is a scenario where a checkbox is marked to delete all rows in a table. We might be able to avoid the log.error by checking whether there are rows in the table. WDYT ?